tests/test_sealed_box.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2013 Donald Stufft and individual contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import, division, print_function
+
+import binascii
+
+import pytest
+
+from nacl.encoding import HexEncoder
+from nacl.public import PrivateKey, PublicKey, SealedBox
+
+
+VECTORS = [
+    # privalice, pubalice, plaintext, encrypted
+    (
+        b"77076d0a7318a57d3c16c17251b26645df4c2f87ebc0992ab177fba51db92c2a",
```

Comment:
For the record, I just tried to insert a vector generated by this driver in VECTORS 
```
    (
        b"886e8e517583c633e93415f5591cc7f85131d74edc1f535fcd3b75d60eb81b9c",
        b"3bae7136e69fa4edb7b56759418e68ec0bba7b80669b889d55cfea2dd82b2362",
        b"7a8b67e028f1c565aa92aee5e144675092b8ef",
        (b"a857f31a360c53ce7407ade7c08de6185dfd402b9a37c2d0d2b3e97931c3c61c1540"
         b"b813f96fb898846a8995f614ae89ad5c0205f6217462316334d09fbe22aa2a7b4a")
    )
```
And the tests are still passing.